src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -95,6 +97,24 @@ public static Email parseEmail(String email) throws ParseException {
         return new Email(trimmedEmail);
     }
 
+    /**
+     * Parses a {@code String gender} into an {@code Gender}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code gender} is invalid.
+     */
+    public static Gender parseGender(String gender, Boolean isNaAllowed) throws ParseException {
+        requireNonNull(gender);
```

Comment:
Perhaps it's good to check that all arguments are non-null just to be safe?
```suggestion
        requireAllNonNull(gender, isNaAllowed);
```